src/lex/token.rs

```diff
@@ -241,16 +241,20 @@ impl<'a> Token<'a> {
                 }
                 wav if wav.starts_with("#WAV") => {
                     let id = command.trim_start_matches("#WAV");
+                    let str = c.next_line_remaining();
                     let filename = Path::new(
-                        c.next_token()
+                        (!(str.is_empty()))
+                            .then(|| str)
                             .ok_or_else(|| c.err_expected_token("key audio filename"))?,
                     );
```

Comment:
```rs
if str.is_empty() {
    return Err(c.err_expected_token("key audio filename"));
}
let filename = Path::new(str);
```
Using a simple `if` could make it more readable. This method chain may be difficult to read.